### ISSUE TITLE
Custom style does not allow entry of specifics = unconfirmed entry

### ISSUE THREAD

**Issue by oshc** [geoffhumphrey/brewcompetitiononlineentry#551]
I added a alcoholic soda category and want to require the special ingredients, carbonation, strength and sweetness be specified. When you add an entry to this category there are no fields to fill out for those specifics. This in turn the site says this is an unconfirmed entry. I figured it would show up like the Berry Mead category for example.

**Comment by geoffhumphrey** [comment]
Confirmed. Extending your "Alcoholic Soda" example, I recreated the problem by:

1. Creating a custom style type called "Soda"
2. Adding a custom style category called "Alcoholic Soda" using the newly created "Soda" style type and making sure that the "Require Special Ingredients" radio is marked "yes."
3. Attempting to add a new entry in the newly created custom style category - the "Special Ingredients" input field does not appear.

A work around until a solution is in place is to reassign the custom category to one of the native style types: Beer, Cider or Mead. If that is done, the special ingredients field does appear when adding an entry. Not ideal, but it will work for now.

Thanks for reporting @oshc!
